src/views/Hooks/ViewHook/index.jsx

```diff
@@ -128,13 +142,13 @@ export default class ViewHook extends Component {
   };
 
   render() {
-    const { isNewHook, data } = this.props;
+    const { isNewHook, hookData, lastFiresData } = this.props;
```

Comment:
One you merge the last fires data in `hook.graphql` then everything will be under `data`.